### PR TITLE
[win32] Fixes for VideoPlayer

### PR DESCRIPTION
**Pull request by afedchin**
also need to rename dvdplayer folder to VideoPlayer inside a few dependencies packages:

fontconfig-2.8.0-win32.7z
freetype-2.4.6-win32-3.7z
gnutls-3.2.3-win32.zip
libass-0.12.1-win32.7z
libbluray-0.8.1-win32-vc120.7z
libexpat_2.0.1-win32.7z
librtmp-20150114-git-a107ce-win32.7z

### CODE REVIEW

**Comment by FernetMenta** [comment]
thanks!

why does fontconfig and friends have a dependency to dvdplayer?

**Comment by FernetMenta** [comment]
those libs are copied to system/players/dvdplayer, right?

**Comment by afedchin** [comment]
Yes, those libs are copied to system/players/dvdplayer or more specifically each package extracted to the root directory and each of them contains system/players/dvdplayer folder so dlls are extracted from those packages to this folder. 
I don't know exactly where fontconfig or freetype are used, IIRC gnutls is requred for ffmpeg, libass for subtitles, libbluray for BluRay's

**Comment by FernetMenta** [comment]
@wsnipex @notspiff do you have a better idea where to copy libs for VideoPlayer. Creating this kind of dependency (system/players/dvdplayer) seems not ideal to me.

**Comment by notspiff** [comment]
they can really be copied wherever we want. but that's why i like the
approach in the cmake buildsystem - out of tree builds. they should not be
copied into the source tree.

**Comment by notspiff** [comment]
see DllPaths_win32.h

File: xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp [hunk 1]
```diff
@@ -94,8 +94,7 @@ void CalculateYUVMatrix(TransformMatrix &matrix
                         , unsigned int  flags
                         , ERenderFormat format
                         , float         black
-                        , float         contrast
-                        , bool          limited)
+                        , float         contrast)
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: I think we should keep limited here. Looks like the signature in the header is wrong
@laurimyllari interesting that it compiles on Linux

Comment (afedchin) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: I agree. Should I fix signature in the header?

Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: yes please

Comment (afedchin) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: done

File: xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp [hunk 2]
```diff
@@ -108,7 +108,7 @@ void CalculateYUVMatrix(TransformMatrix &matrix
       coef.m[row][col] = conv[col][row];
   coef.identity = false;
 
-  if(g_Windowing.UseLimitedColor() || limited)
+  if(g_Windowing.UseLimitedColor())
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: now we should keep this, right?

Comment (afedchin) [inline xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp]: right. updated.

**Pull request merged**